src/containers/User/AddUser/index.js

```diff
@@ -0,0 +1,168 @@
+import React from  'react';
+import {
+  Button,
+  Card,
+  Checkbox,
+  Col,
+  Form,
+  Input,
+  Row,
+  Select,
+  Switch,
+  Typography,
+} from 'antd';
+import { PlusOutlined } from '@ant-design/icons';
+import PropTypes from 'prop-types';
+
+import styles from './style.module.css';
+
+const { Title } = Typography;
+const { Option } = Select;
+const requiredMessage = 'Este campo é obrigatório!';
+
+const formRequireRules = [{
+  message: requiredMessage,
+  required: true,
+}]
+
+const AddUser = ({
+  allowAddTypeAccount,
+  allowCustomPermissions,
+  form,
+  goToAddTypeAccount,
+  handleAllowSetCustomPermissions,
+  handleOnTypeAccountChange,
+  handleSubmit,
+  permissions,
+  typeAccounts,
+}) => {
+    const renderCheckbox = ({
+      label,
+      name,
+      value,
+    }) => (
+      <Col
+        className={styles.checkBoxheight}
+        span={12}
+        key={name}
+      >
+        <Form.Item
+          name={name}
+          valuePropName="checked"
+        >
+          <Checkbox
+            disabled={!allowCustomPermissions}
+            value={value}
```

Comment:
usar prop "checked" para especifica se o Checkbox está selecionado, conforme a documentação especifica e conforme você declarou em na propriedade valuePropName do <Form.Item> pai deste Checkbox
```suggestion
            checked={value}
```